### DISCUSSION TITLE
createArrowFiles but with differing filtering criteria

### DISCUSSION THREAD

**Discussion by poorvi4** [GreenleafLab/ArchR#1569] [Questions / Documentation]
To whom it may concern,

Thanks for the wonderful package!  I'm trying to automate the process of creating arrow files for any given list of fragment files inputed into my script.  I was wondering if it were possible to pass differing minTSS and minFrags criteria corresponding to each fragments file in the inputFiles list? I tried doing this with calling createArrowFiles(...) in lapply for each fragments file and their respective QC criteria but realized this was not feasible given the hdf5 format. Is there a workaround for this aside from explicitly calling the function multiple times for each file?

Once again thank you for the package!

**Comment by rcorces** [comment]
You could use a standard cutoff, create arrow files, and then for each sample/Arrow File identify the cells you want to keep, keep track of all of those cell names, and then pass them to `subsetArchRProject()` to create new copies of the Arrow files with only the cells you desire. not sure that helps but other than calling `createArrowFiles()` individually, I think thats the best way.
